src/raftstore/store/store.rs

```diff
@@ -1392,6 +1392,17 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         STORE_SNAPSHOT_TRAFFIC_GAUGE_VEC.with_label_values(&["receiving"])
             .set(snap_stats.receiving_count as f64);
 
+        let mut apply_snapshot_count = 0;
```

Comment:
It's hard to control the applying process. Snapshot is probably applied before store heartbeat is about to be collected.